config/helm/chart/default/templates/Common/csi/daemonset.yaml

```diff
@@ -272,7 +272,5 @@ spec:
           operator: Exists
           effect: NoSchedule
   updateStrategy:
```

Comment:
when setting `updateStrategy: ` or `updateStrategy: {}` in the values file I get the following generated yaml for the daemonset:
```yaml
  updateStrategy:
    null
```
and
```yaml
  updateStrategy:
    {}
```

They both seem to lead to the following upgrade strategy in the actual daemonset: 
```yaml
  updateStrategy:
    rollingUpdate:
      maxSurge: 0
      maxUnavailable: 1
    type: RollingUpdate
```

This is functionally equivalent to what we are explicitly setting (default value of `maxSurge` is 0), so my question would be: do we even need the default value in the values file?

Reply:
I prefer to have the defaults set explicitly

Reply:
can do